service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature

```diff
@@ -85,6 +86,50 @@ Feature: AIS
       | DE11760365688833114935;DE13760365689669622432 | DE11760365688833114935 | DE11760365688833114935 | PSU-Successful | 12345    | SMS_OTP    | 54321 |
       | DE11760365688833114935                        | null                   | null                   | PSU-Successful | 12345    | SMS_OTP    | 54321 |
 
+    ################################################################################################
+    #                                                                                              #
+    # Get Transactions                                                                             #
+    #                                                                                              #
+    ################################################################################################
+  Scenario Outline: Get Transaction List
+    Given PSU created a consent on dedicated accounts for account information <accounts>, balances <balances> and transactions <transactions>
+    And PSU authorised the consent with psu-id <psu-id>, password <password>, sca-method <sca-method> and tan <tan>
+    When PSU accesses the account list
+    When PSU accesses the transaction list
+    Then the transaction list data are received
+    Examples:
+      | accounts                                      | balances               | transactions           | psu-id         | password | sca-method | tan   |
+      | DE11760365688833114935;DE13760365689669622432 | DE11760365688833114935 | DE11760365688833114935 | PSU-Successful | 12345    | SMS_OTP    | 54321 |
+      | DE11760365688833114935                        | null                   | null                   | PSU-Successful | 12345    | SMS_OTP    | 54321 |
+
+  Scenario Outline: Get Single Transaction
+    Given PSU created a consent on dedicated accounts for account information <accounts>, balances <balances> and transactions <transactions>
+    And PSU authorised the consent with psu-id <psu-id>, password <password>, sca-method <sca-method> and tan <tan>
+    When PSU accesses the account list
```

Comment:
only use a single "When" step. Move the other steps into the given statements:

-> Given ...
And ...
And PSU accesse**d** the account list
And ...
When PSU accesses a single transaction

Reply:
Resolved the multiple When steps and renamed it to ANDs.

We commited to not rename **accesses**, because it is used in another scenario as When step.